lib/tasks/import_designations.rake

```diff
@@ -29,13 +31,32 @@ namespace :stockit do
         order.country_id = Country.find_by(stockit_id: value["country_id"]).try(:id)
       end
 
-      order.status = value["status"]
+      order.state = case value["detail_type"]
+                    when "CarryOut", "Shipment"
+                      Order::SHIPMENT_STATUS_MAP[value["status"]]
+                    when "LocalOrder"
+                      Order::LOCAL_ORDER_STATUS_MAP[value["status"]]
+                    else
+                      value["status"].try(:downcase)
+                    end
+
       order.code = value["code"]
       order.detail_type = value["detail_type"]
       order.created_at = value["created_at"]
-      order.description = value["description"]
+      order.updated_at = value["updated_at"] if value["updated_at"].present?
+      order.people_helped = value["number_of_people_helped"]
+      order.continuous = value["continuous"]
+      order.shipment_date = value["sent_on"]
+      order.staff_note = value["comments"]
+      order.purpose_description = value["description"]
+
+      %w[process_completed_by_id cancelled_by_id closed_by_id dispatch_started_by_id
```

Comment:
technically not every order imported can be cancelled and closed. I wonder if we should omit these `_by_id` fields instead so that they are null. I think that would be fine as they make no sense in a post-stockit system. So could you remove this block of code?

Reply:
Sure, Makes Sense. :+1: